Core/src/com/serotonin/m2m2/rt/maint/work/BackupWorkItem.java

```diff
@@ -112,7 +114,7 @@ public void execute() {
             LOG.info("Starting backup WorkItem.");
             // Create the filename
             String filename = "Mango-Configuration";
-            String runtimeString = new SimpleDateFormat(BACKUP_DATE_FORMAT).format(new Date());
+            String runtimeString = DateTimeFormatter.ofPattern(BACKUP_DATE_FORMAT).format(LocalDateTime.now());
```

Comment:
I'm with Jared on this, the format didn't change so I can't see how the resulting filename would be any different.  Did you actually test this by generating a backup?

Also if we do change the format, we will have the problem that the old backups may not be importable/usable by Mango so you would need to look into that.

Assuming the restore logic uses the same format the best solution I could see is to make this a mango.property and let our customer's choose what pattern they want.

Reply:
Hi the format it is not changing. What it is changing is that instead of creating an instance of Date class now we will get the current LocalDateTime. So the file name remains the same. I test it locally that this DateTimeFormatter retrieves the same date string

Reply:
The sample name is 

`Mango-Configuration-Feb-07-2023_135404.json`

The date it is just used when there are more than 1 instance:

`int maxFiles = SystemSettingsDao.getInstance().getIntValue(SystemSettingsDao.BACKUP_FILE_COUNT);
            // If > 1 then we will use a date in the filename
            if (maxFiles > 1) {`